libbeat/docs/highlights-7.0.0.asciidoc

```diff
@@ -0,0 +1,104 @@
+developed an upgrade procedure that uses {es} field aliases to make the
+transition easier. After the upgrade is complete, we strongly advise that you
+adjust your custom {kib} dashboards, machine learning jobs, and other content to
+use the new ECS field names. 
+
+See the {beats-ref}/upgrading.html[{beats} upgrade documentation] for more
+information.
+
+[float]
+==== Index lifecycle management (ILM)
+
+In 6.6, {es} added advanced capabilities for index management. Rather than
+simply performing management actions on your indices on a set schedule, you can
+base actions on other factors such as shard size and performance requirements.
+You control how indices are handled as they age by attaching a lifecycle policy
+to the index template used to create them. You can update the policy to modify
+the lifecycle of both new and existing indices. This set of capabilities are
+grouped in the {ref}/index-lifecycle-management.html[index lifecycle management
+(ILM)] APIs.
+
+In 7.0, {beats} defaults to rotating indices by using ILM policies, if the {es}
+version to which they connect supports ILM. The default policy rotates indices
+when they reach 50 GB or 30 days. You can edit the ILM policy by using the {kib}
+management UI, or directly via the {es} API.
+
+[float]
+==== Stack monitoring
+
+The full suite of modules to {stack-ov}/monitoring-production.html[monitor your
+{stack}] are now GA. These include the {metricbeat} modules for {es}, {ls}, and
+{kib}.
+
+In the future, we will switch to {metricbeat} as the recommended agent
+for monitoring the {stack}. To prepare for the switch, see
+{ref}/configuring-metricbeat.html[Collecting {es} monitoring data with {metricbeat}].
+
+[float]
+==== Logs and infrastructure metrics
+
+{beats} adds several new modules, focusing on datastores and the cloud.
+
+On the cloud side, {metricbeat} adds the
+{metricbeat-ref}/metricbeat-module-aws.html[AWS] module, which collects and
+centralizes basic resource utilization metrics from all your EC2 instances,
+directly from Cloudwatch. A widely used messaging platform,
+{metricbeat-ref}/metricbeat-module-nats.html[NATS], earns its own module for
+capturing stats, connections, routes, and subscriptions metrics.
+
+For datastores, {metricbeat} offers modules for Microsoft SQL Server and
+CouchDB. The {metricbeat-ref}/metricbeat-module-mssql.html[MSSQL] module
+captures transaction log and performance counters, while the
+{metricbeat-ref}/metricbeat-module-couchdb.html[CouchDB] module provides a
+server metricset.
+
+[float]
+==== Security analytics data sources
+
+For data relevant to security analytics, {filebeat} adds a
+{filebeat-ref}/filebeat-module-zeek.html[Zeek] module that integrates with the
+popular open-source Zeek project, formerly known as Bro, and a
+{filebeat-ref}/filebeat-module-santa.html[Santa] module, which tracks process
+executions on macOS. These modules add to the list of data sources already
+supported in the 6.x series, including
```

Comment:
Is there a reason this sentence differs from the original PR? In the original PR it says:

> ... list of modules added already in the 6.x series, ...

Whereas over here in this PR the same sentence says:

> ... list of data sources already supported in the 6.x series, ...

Reply:
Ah never mind, I just read your PR description, specifically:

> I've changed the language to indicate that NetFlow is a supported input.

And I noticed the commit that goes with this change as well: 874d70a03ef7c3a5fdf3f533d503b5ac4eb70aa2.

So all good here!

Reply:
@ycombinator yup! Because the NetFlow module is not available in the 6.x series, nor is it in 7.0. It's supported as an input in those releases.